snaps/features/data-storage.md

```diff
@@ -31,7 +30,7 @@ Add the following to your Snap's manifest file:
 }
 ```
 
-### 2. Use encrypted storage
+## Use encrypted storage
```

Comment:
Why remove the numbering? In the previous it is included.

Reply:
Because you might choose to use either encrypted or unencrypted storage; they are not both required as "steps." This is for consistency as the dialogs guide is similarly unordered.